Index: configure
===================================================================
--- configure
+++ configure	2019-10-20 19:58:17.082053402 +0200
@@ -2325,7 +2325,7 @@
 void _ebcdic() { char* s = (char*) ebcdic_mm; s = (char*) ebcdic_ii; }
 int main() { _ascii (); _ebcdic (); return 0; }
 EOF
-$CXX $LDFLAGS $CXXFLAGS -o $TMPO$HOSTEXEEXT tmp_endianness_check.cpp
+$CXX $CXXFLAGS -fno-lto -c -o $TMPO.o tmp_endianness_check.cpp
 if $_strings $TMPO$HOSTEXEEXT | grep BIGenDianSyS >/dev/null; then
 	_endian=big
 elif $_strings $TMPO$HOSTEXEEXT | grep LiTTleEnDian >/dev/null; then
